pkg/audit/verifier.go

```diff
@@ -71,12 +72,14 @@ func (verifier *Verifier) Verify(ctx context.Context, stripe *Stripe) (verifiedN
 
 	var offlineNodes storj.NodeIDList
 	var failedNodes storj.NodeIDList
+	containedNodes := make(map[int]storj.NodeID)
 	sharesToAudit := make(map[int]Share)
 
 	for pieceNum, share := range shares {
 		if shares[pieceNum].Error != nil {
+			// TODO(kaloyan): we need to check the logic here if we correctly identify offline nodes from those that didn't respond.
 			if shares[pieceNum].Error == context.DeadlineExceeded || !transport.Error.Has(shares[pieceNum].Error) {
-				failedNodes = append(failedNodes, nodes[pieceNum])
+				containedNodes[pieceNum] = nodes[pieceNum]
```

Comment:
I understand why this addition is here, but maybe we should wait until we add more tests and implement the way for pending audits to be removed from the containmentDB before putting them in? it might not be a big deal if we add them now though.

Reply:
Even if we deploy containment mode semi-ready like in this PR, it still won't be worse than the current situation for storage nodes. Here we put nodes in containment mode rather than failing the audit for them. So we won't hurt unnecessarily their reputations